drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -279,7 +279,7 @@ function DragNDropTemplates(url_name) {
                     problemHeader,
                     h('p', {innerHTML: ctx.problem_html}),
                 ]),
-                h('section.drag-container', { attributes: { role: 'application' } }, [
+                h('section.drag-container', { attributes: { } }, [
```

Comment:
@itsjeyd No good reason no.. removed with [b5ab955](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/67/commits/b5ab955f96ef1adb6d14699597872ad705f3355c).